mmdet/apis/inference.py

```diff
@@ -202,3 +207,54 @@ async def async_inference_detector(model, imgs):
     torch.set_grad_enabled(False)
     results = await model.aforward_test(data, rescale=True)
     return results
+
+
+def show_result_pyplot(model: BaseDetector,
```

Comment:
```suggestion
def show_result(model: BaseDetector,
```

Reply:
What I'm thinking is whether this function should only support pypolt? If your purpose is to only support pypolt, then you really don't need to consider general

Reply:
Maybe you can take a look first https://github.com/open-mmlab/mmdetection/pull/8975/files